.github/workflows/test.yml

```
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  pull_request:
+  push:
```

Perhaps better to add back before merging:

```suggestion
  push:
    branches: [main]
```